lib/helpers.js

```diff
@@ -131,3 +132,77 @@ exports.getBaseModelForDataSourceName = function(name, allDataSources) {
   return connectorMeta && connectorMeta.baseModel;
 };
 
+/**
+ * Get the available type choices that should be used for arguments
+ * from workspace.
+ */
+exports.getTypeChoices = function() {
+  var typeChoices = ModelProperty.availableTypes.concat({
+    name: '(other)',
+    value: null
+  });
+  return typeChoices;
+};
+
+/**
+ * Print a sample dummy method implemented in `modelName.js` file
+ * @param {String} modelName Name of model
+ * @param {String} methodName Name of remote method
+ * @param {Boolean} isStatic Method defined on constructor with value true
+ *  while defined on prototype with value false
+ * @param {Array} accepts An array of accept arguments
+ * @param {Array} returns An array of return arguments
+ */
+exports.printDummyMethod = function(
+  modelName,
+  methodName,
+  isStatic,
+  accepts,
+  returns
+) {
+  String.prototype.splice = function(idx, str) {
+    var len = this.length;
+    return this.slice(0, len-idx) + str + this.slice(len-idx);
+  };
+
+  var comma = ', ';
+  var acceptArg = '';
+  var returnArg = '';
+  var functionDef = '= funtion (callback) {' + '\n' +
+  '  //TODO \n' +
+  '  callback(null' + '); \n'+
+  '}';
+  var prototype = '.prototype';
+
+  functionDef = methodName.concat(functionDef);
+  functionDef = '.'.concat(functionDef);
+  if(!isStatic) functionDef = prototype.concat(functionDef);
+  functionDef = modelName.concat(functionDef);
+
+  if(accepts.length>0) {
+    accepts.forEach(function(accept) {
+      acceptArg = acceptArg.concat(accept.arg);
+      acceptArg = acceptArg.concat(comma);
+    });
+    functionDef = functionDef.splice(42, acceptArg);
+  }
+
+  if(returns.length>0) {
+    var varDeclare = '  var ' + ';\n';
+    var lastReturn = returns.pop();
+    functionDef = functionDef.splice(30, varDeclare);
+    returns.forEach(function(_return) {
+      returnArg = returnArg.concat(_return.arg);
+      returnArg = returnArg.concat(comma);
+    });
+    returnArg = returnArg.concat(lastReturn.arg);
+    functionDef = functionDef.splice(32, returnArg);
+    returnArg = comma.concat(returnArg);
+    functionDef = functionDef.splice(5, returnArg);
+  }
+
+  console.log('This is a sample dummy method. Please implement it in ' +
+    modelName +
+    '.js:');
```

Comment:
- let's rephrase this sentence a bit:
    ```
    We have added strong-remoting metadata for your new method to %s.
    Now it's up to you to provide implementation in %s.
    Here is a sample code to get you started:

    MyModel.mymethod = function(callback) {
      // TODO
      callback(null);
    };
    ```

- use full relative path, not just a file name, i.e. `common/mymodel.json` and `common/mymodel.js`, where `common` is the *facet* name (you can get it from a property of `ModelDefinition`).
- use Yeoman's logging methods instead of `console.log`, consider adding some colours
- add an empty line to visually delimit the text from the method body